hack/systemd_tag.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+if pkg-config --exists libsystemd; then
```

Comment:
does this work on ubuntu too?  i think pkg-config calls and their names (libsystemd) are distro-independent correct?

Reply:
I tested it with Ubuntu 18.04.2 LTS and the package `libsystemd-dev`. It works.